src/main/java/uk/gov/hmcts/reform/blobrouter/data/notifications/NotificationEnvelopeRepository.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.blobrouter.data.notifications;
+
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class NotificationEnvelopeRepository {
```

Comment:
maybe this class should be renamed as well

Reply:
Renamed